eth/backend.go

```diff
@@ -141,6 +141,10 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 	if err := pruner.RecoverPruning(stack.ResolvePath(""), chainDb, stack.ResolvePath(config.TrieCleanCacheJournal)); err != nil {
 		log.Error("Failed to recover state", "error", err)
 	}
+	if config.RPCGasInflationRate == 0 {
+		// if it was not set, default it as 1
+		config.RPCGasInflationRate = 1
```

Comment:
does this mean that it's 1 by default or 1.3 as below?

Reply:
it's 1.3 by default because we set all the defaults with flags. This is more a net just in case someone builds a node object by their own. In that case, as the defaults are only set as part of reading flags, the rate will be in 0 which is not good at all.
I didn't want to change how we handle the defaults, just made the same as others, and found this possible scenario